config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
```

Comment:
The password route is also enabled. All of these are devise default. The number of modules enabled does not correlate to the number of routes.